### DISCUSSION TITLE
motif enrichment analysis

### DISCUSSION THREAD

**Discussion by urnotchris1** [GreenleafLab/ArchR#1707] [Questions / Documentation]
Hi, I want to use the peakAnnoEnrichment() function to plot motif-enrichment volcano , but there are no fold change values in the data generated by this function.  I am wondering how I can do this.
thank you!

**Comment by rcorces** [comment]
This is not possible. This function returns enrichment from a significance test. There is no fold-change associated with this metric.

**Reply by urnotchris1** [reply]
Thank you very much for your answer. I would like to know how to do it if I want to annotate the 
 motifs that are differentially enriched in one cluster relative to another cluster in the volcano chart.
